File: examples/petstore/oseg_generated/php/DeleteOrder_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    (new OpenAPI\Client\Api\StoreApi($config))->deleteOrder(
        order_id: "12345",
    );
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling Store#deleteOrder: '
        . print_r($e->getResponseObject());
}

File: examples/petstore/oseg_generated/csharp/DeleteOrder_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class DeleteOrderDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            new StoreApi(config).DeleteOrder(
                orderId: null
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Store#DeleteOrder: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
